determined-ai/determined
# chore: introduce `Prettier`

## Description
[DET-8097]



## Test Plan
- Check if all rules are good for all
- Check if prettier works on local
- Check if CI catches perttier error 



## Commentary (optional)
- husky and lint-staged are not included in this PR
- I adjusted some rules to standard rules, such as space in array `[ 1, ,2, 3 ]` -> `[1, 2, 3]`



## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[DET-8097]: https://determinedai.atlassian.net/browse/DET-8097?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
